prince-of-versions/src/main/java/co/infinum/princeofversions/helpers/parsers/JsonVersionConfigParser.java

```diff
@@ -140,7 +152,8 @@ public VersionContext parse(JSONObject data) throws JSONException {
                     this.currentVersion,
                     new VersionContext.Version(minVersion.toString()),
                     currentVersion.lessThan(minVersion),
-                    new VersionContext.UpdateContext(new VersionContext.Version(latestVersion.toString()), notificationType),
+                    new VersionContext.UpdateContext(new VersionContext.Version(latestVersion.toString()), notificationType, lastMinSdk,
```

Comment:
I think that `lastMinSdk` has nothing to do with `UpdateContext`. `lastMinSdk` is related with `minVersion` and should be in root object. Also, please make `optionalUpdate` field in `VersionContext` nullable so it is clear if someone is using it without null check.